worker/echo/com.redhat.Yggdrasil1.Worker1.echo.conf

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!DOCTYPE busconfig PUBLIC
+ "-//freedesktop//DTD D-BUS Bus Configuration 1.0//EN"
+ "https://dbus.freedesktop.org/doc/busconfig.dtd">
+ <busconfig>
+ <policy user="root">
```

Comment:
Is root user really necessary?

Reply:
This bus policy only applies to the service when running on the system bus. The current implementation assumes that both yggd and workers run as root, so the policy is applied to the root user.